### ISSUE TITLE
IRC client.dart not working in Flutter 1.9.1

### ISSUE THREAD

**Issue by jspablo** [SpinlockLabs/irc.dart#32]
I get the following error when running the [Getting started](https://github.com/SpinlockLabs/irc.dart/wiki/Getting-Started) tutorial:

``` bash
Compiler message:
../../../../development/flutter/.pub-cache/hosted/pub.dartlang.org/irc-4.0.1/lib/src/client/connection.dart:53:28: Error: The argument type 'Utf8Decoder' can't be assigned to the parameter type 'StreamTransformer<Uint8List, dynamic>'.
 - 'Utf8Decoder' is from 'dart:convert'.
 - 'StreamTransformer' is from 'dart:async'.
 - 'Uint8List' is from 'dart:typed_data'.
Try changing the type of the parameter, or casting the argument to 'StreamTransformer<Uint8List, dynamic>'.
          .transform(const Utf8Decoder(allowMalformed: true))
```

I couldn't fix it with the suggested solution, seems related to [this](https://groups.google.com/forum/#!topic/flutter-announce/LTe4SYU8-0Q)

Any help is appreciated

Thank you!

**Comment by kendfinger** [comment]
Hi @jspablo, this has been fixed in v4.0.2 of the library. Thanks!

**Comment by jspablo** [comment]
So fast! Thank you!
